comp/api/api/apiimpl/server_cmd.go

```diff
@@ -88,7 +88,7 @@ func startCMDServer(
 	pb.RegisterAgentServer(s, &server{})
 	pb.RegisterAgentSecureServer(s, &serverSecure{
 		configService: configService,
-		taggerServer:  taggerserver.NewServer(tagger.GetDefaultTagger()),
```

Comment:
can we add the tagger.Component as parameter of this `startCMDServer()`  function?